### DISCUSSION TITLE
Relative Url with functional search for IPFS deployment or local

### DISCUSSION THREAD

**Discussion by kmlbgn** [imfing/hextra#250] [Ideas]
Hi there,

I've tweaked the theme for better compatibility with Hugo's `relativeUrls` parameters, particularly for Flexsearch support in this setting. `relativeUrls` is really useful for IPFS deployment for instance.

However, full local browsing using `file://` protocol isn't smooth yet. It's hung up on the script tag integrity checks for assets which are created in production environment (`hugo.IsProduction`). Flipping environments doesn't solve it completely – because some integrity checks are outside this conditional.

Keen to get your take on this before. Would you have any interest in having this feature for the theme?

**Comment by imfing** [comment]
@kmlbgn A solution could be overriding the following template to remove the tag integrity:

https://github.com/imfing/hextra/blob/main/layouts/partials/scripts.html

> Keen to get your take on this before. Would you have any interest in having this feature for the theme?

Did you mean adding a flag to enable/disable the data integrity for the script tags?
